docs/documentation_styleguide.rst

```diff
@@ -0,0 +1,234 @@
+========================
+Documentation Styleguide
+========================
+
+.. topic:: Table of contents
+
+    A Table of Contents that should be shown different to admonitation.
+
+.. admonition:: Description
+
+    A guide to write Documentation for Plone and for Plone Add-ons
+
+
+Introduction
+============
+
+Having a 'best practices' approach for writing your documentation will benefit the users and the community at large.
+
+Even better: when there is a clear structure and style for your documentation, the chances that other people will help improve the documentation increase!
+
+Further advantages of following this guide:
+
+* The documentation can be included on `docs.plone.org <http://docs.plone.org>`_
+* It will be in optimal format to be translated with tools like `Transifex <https://www.transifex.com/>`_.
+
+Tone
+----
+
+Guides should be informational, but friendly. Use the active voice whenever possible, and contractions and pronouns are acceptable (in particular, the use of you in regards to the reader).
+
+Use common sense – if a term is related to a high-level concept that fewer people would know, then take a sentence or two to explain it.
+
+**Your documentation is not code.**
+
+It needs to be translatable. No, not into PHP, but into Chinese, Catalan, Klingon, ...
+
+Think about it this way:
+
+Each sentence in the documentation can be turned into a .po string.
+Breaking sentences with linebreaks would mean a translator will only see part of the sentence, making it impossible to translate.
+
+Documentation Structure & Styleguide
+====================================
+
+For including documentation into docs.plone.org, **please** follow these guidelines:
+
+
+* All documentation should be written in **valid** `ReStructuredText <http://docutils.sourceforge.net/rst.html>`_  There are some :doc:`helper_tools` available.
+
+* The top level of your package should contain the following documentation-related files:
+
+  - README.rst   This should be a **short** description of your add-on, not the entire documentation!
+    See the :ref:`styleguide-readme-example`
+
+  - CHANGES.rst  This should track the feature changes in your add-on, see :ref:`styleguide-changes-example`
+
+  - CONTRIBUTORS.rst  This should list the people writing, translating and otherwise contributing
+
+* All of your (longer) end-user documentation should go into the /docs subdirectory. Feel free to split your documentation into separate files, or even further subdirectories if it helps clarity.
+
+* Make **absolutely** sure there is a start page called index.rst.
+  It is also usually a *really* good idea to have that include a Table of Content, see :ref:`styleguide-toc-example`
+
+* use relative links for internal links within your /docs/ directory, to include images for instance.
```

Comment:
Are we going to use some reST markup to indicate that "/docs/" and other directories and filenames are names?